[2024-07-13T04:07:26.502+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T04:07:26.520+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T04:07:09.436595+00:00 [queued]>
[2024-07-13T04:07:26.525+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T04:07:09.436595+00:00 [queued]>
[2024-07-13T04:07:26.526+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-13T04:07:26.541+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-07-13 04:07:09.436595+00:00
[2024-07-13T04:07:26.549+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2263) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-13T04:07:26.551+0000] {standard_task_runner.py:63} INFO - Started process 2265 to run task
[2024-07-13T04:07:26.552+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***_book_api_pipeline', 'transform_data', 'manual__2024-07-13T04:07:09.436595+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/***_book_api_dags.py', '--cfg-path', '/tmp/tmpcxq0_b83']
[2024-07-13T04:07:26.552+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask transform_data
[2024-07-13T04:07:26.599+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow_book_api_pipeline.transform_data manual__2024-07-13T04:07:09.436595+00:00 [running]> on host e6b306b9c204
[2024-07-13T04:07:26.686+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zainal' AIRFLOW_CTX_DAG_ID='***_book_api_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-13T04:07:09.436595+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-13T04:07:09.436595+00:00'
[2024-07-13T04:07:26.688+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T04:07:26.871+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-13T04:07:26.872+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T04:07:26.879+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***_book_api_pipeline, task_id=transform_data, run_id=manual__2024-07-13T04:07:09.436595+00:00, execution_date=20240713T040709, start_date=20240713T040726, end_date=20240713T040726
[2024-07-13T04:07:26.925+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-13T04:07:26.951+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-13T04:07:26.961+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
